app/page.tsx

```diff
@@ -25,6 +25,7 @@ import { useStep } from "./context/StepContext";
 import { clearFormState } from "./utils";
 import { trackEvent } from "./hooks/analytics";
 import { useNetwork } from "./context/NetworksContext";
+import { useRouter } from "next/router";
```

Comment:
this import will give you this error:

```
 ⨯ Error: NextRouter was not mounted. https://nextjs.org/docs/messages/next-router-not-mounted
    at Home (./app/page.tsx:45:75)
```

check the next.js docs for how to fix: https://nextjs.org/docs/messages/next-router-not-mounted